articles/Choosing-between-Django-Flask-and-FastAPI/index.md

```diff
@@ -0,0 +1,79 @@
+
+#### Flask
+It is a micro web framework written in Python. It is built to make building web apps with Python easy and fast and scales up very complex applications. It was more like an April fool’s prank by the creator Armin Ronacher on the 1st of April, where he wrapped Werkzeug and Jinja, which were both created by him. Surprisingly, it was appreciated, which made it to be taken seriously, and more work was done on it.
+
+##### Pros
+1. Flexible and Easy: Most of the parts of Flask have the possibility of changing, which is very unlikely for some other web frameworks.
+2. Testing: Flask allows unit testing and would enable you to transit to a web framework by tweaking some extensions because of its built-in-development server, integrated support, etc.
+3. Easy to Understand: Flask is pretty much very beginner-friendly because of its simplicity, giving room to learn and understand it better. It also enables devs to create apps effortlessly quickly. 
+
+#### Cons
+1. Modules: Flask uses Modules, a third-party involvement prone to cause security breaches. The modules are now like a third-party between the framework and the developer.
+2. Scale: Flask has a singular source that implies that it will handle every request in turns, one after the other, so regardless of how many multiple requests it still takes them in turns, it takes more time.
+
+##### Flask: Use cases
+Flask can be used for commercial projects. It can help you get started quickly but doesn’t work well for resembling a real load. You can easily implement projects with Flask: E-commerce systems, Facebook/ Twitter bot, An online social network, static sites, etc. Still, it’s not advisable to be used for high-load enterprise software.
+
+#### FastAPI  
+Fast API is a modern open-source fast and highly performant Python web framework for building Web APIs and is based on Python 3.6+ standard type hints. 
+
+##### Pros
+1. Data validation: Fast API validates the developer’s data type even in deeply nested JSON requests.
+2. Standards: Fast API is built on standards like JSON Schema, OAuth 2.0 and OpenAPI
+3. GraphQL support: FastAPI makes it easy to build a GraphQL API with a Python library called *graphene-python*.
+
+#### Cons
+1. Small Community: Because FastAPI is relatively new, the community is small compared to other frameworks
+2. Limited Educational materials: Since FastAPI provides detailed documentation, there are very few external educational materials.
+
+##### FastAPI Use cases
+Fast API, which is relatively new amongst the three, works perfectly if your concern is speed. Netflix uses it for its internal crisis management. It also scales perfectly in deploying production-ready machine learning models because ML models work best in production when they are wrapped around a REST API and deployed in a microservice.
+
+Now we’re going to compare Django, Flask, and FastAPI based on their *packages*, *community*, and *performance*.
```

Comment:
I feel there is content lacking here. Can you add additional comparison parameters? That way, it would help the reader choose better.